eng/.emscripten

```diff
@@ -1,9 +1,10 @@
 import os
 
-LLVM_ROOT = os.path.expanduser(os.getenv('DOTNET_EMSCRIPTEN_LLVM_ROOT', ''))
-NODE_JS = os.path.expanduser(os.getenv('DOTNET_EMSCRIPTEN_NODE_JS', ''))
-BINARYEN_ROOT = os.path.expanduser(os.getenv('DOTNET_EMSCRIPTEN_BINARYEN_ROOT', ''))
```

Comment:
we're setting these here, I don't think we can just remove them: https://github.com/dotnet/runtime/blob/28c4dce2c6a9a3619faa612095ed2125d03fbecd/src/mono/browser/build/BrowserWasmApp.targets#L202-L204

i.e. we need to make sure the toolchain still works when using the nuget packages on end-user machines.
I'm not sure everything uses the emsdk_env scripts for example.

Reply:
In the current state we are overwriting the contents of this file anyway, during provisioning, before it gets used.

> i.e. we need to make sure the toolchain still works when using the nuget packages on end-user machines.
I produced nugets in this repo, copied them to runtime repo and tested how the build process goes and it worked fine. If that succeeded, where else it could go wrong? What other tests do you propose?

Reply:
The runtime build is kinda special. I'd recommend trying a normal blazor wasm build.

Reply:
Thanks, tested blazor:
- run
- publish
- AOT publish
No errors detected.